adapters/handlers/rest/configure_api.go

```diff
@@ -429,6 +439,15 @@ func configureAPI(api *operations.WeaviateAPI) http.Handler {
 		ctx, cancel := context.WithTimeout(context.Background(), 60*time.Second)
 		defer cancel()
 
+		if telemetryEnabled(appState) {
+			// must be shutdown before the db, to ensure the
+			// termination payload contains the correct
+			// object count
+			if err := telemeter.Stop(ctx); err != nil {
+				panic(err)
```

Comment:
I'm not sure telemeter's failure in sending data should impact database's graceful shutdown.

Reply:
Should telemeter's Stop be blocking operation? Waiting here for data to be sent reduces context's remaining time to be used on shutting db.

Maybe data required for termination payload could be collected synchronously (before db will be shutdown) while sending could be performed asynchronously in non blocking manner.

Reply:
Agreed, I've refactored to make the telemetry stop:
1. Not prohibit the rest of the shutdown with a panic
2. Use a different deadlined context that doesn't affect the rest of the DB 60sec shutdown